CHANGELOG.md

```diff
@@ -2,9 +2,44 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.9 September 22, 2021
+# 10.0.0 October 6, 2021
 
-**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+**The Mapbox Maps SDK for Android is now ready for production use.**
+
+## Breaking changes ⚠️
+* Add `@JvmOverloads` where applicable to provide better experience for Java users. ([#656](https://github.com/mapbox/mapbox-maps-android/pull/656))
+* Refactor gestures configuration options to be aligned better across platforms. ([#672](https://github.com/mapbox/mapbox-maps-android/pull/672))
```

Comment:
shall we include the list of gestures changes in the change log? or is it better to be in the release notes?